### DISCUSSION TITLE
Welcome to redux-devtools Discussions!

### DISCUSSION THREAD

**Discussion by Methuselah96** [reduxjs/redux-devtools#848] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Obiwarn** [comment]
Can I change the blue arrow symbol somehow?



<img width="38" alt="image" src="https://github.com/reduxjs/redux-devtools/assets/9539797/5a5ef690-5e0b-4972-9604-81d87aba1456">

ARROW_COLOR sadly does not change the symbol in Chrome. Here I set it to black (arrow still blue):

<img width="237" alt="image" src="https://github.com/reduxjs/redux-devtools/assets/9539797/3e15b16b-4dab-4484-b425-70b8dde91690">
